### ISSUE TITLE
Application Fails

### ISSUE THREAD

**Issue by davidcayll** [beatngu13/pdf-zoom-wizard#20]
Running Java 8 Update 201 on Windows 7 32-bit OS.  Downloaded versions 0.1.1 and 0.1.2 and placed in C: folder.  Both versions report FAILED after running for half a second.  Double clicked the .jar files as per instructions and completed Browse to single file, click RUN and accept overwrite of file.  What am I doing wrong?

**Comment by beatngu13** [comment]
This is likely related to #17. Could you attach a sample PDF, so I can reproduce the error?

**Comment by beatngu13** [comment]
Hey @davidcayll,

thank you for your [helpful feedback](https://github.com/beatngu13/pdf-zoom-wizard/issues/17#issuecomment-460278809). I was able to reproduce the issue. The problem is that the bookmark "MA - MAINTENANCE" seems to be broken. When you click on it, it doesn't jump to a specific target. This causes an issue in the underlying library I'm using for accessing the PDF.

I came up with a small fix, which seems to sidestep the issue. Will try to release this as part of v0.2.0, probably next week.

Regarding:

>Maybe error handling messages would be handy to include in the final application?

Definetly, already working on better log messages.

Best,
Daniel
